speech/google/cloud/speech/operation.py

```diff
@@ -123,8 +123,9 @@ def _update(self, response):
         raw_results = response.get('response', {}).get('results', None)
         results = []
         if raw_results:
-            for result in raw_results[0]['alternatives']:
-                results.append(Transcript.from_api_repr(result))
+            for result in raw_results:
+                for alternative in result['alternatives']:
+                    results.append(Transcript.from_api_repr(alternative))
```

Comment:
@daspecster I finally understand what you were worried about: a [list][1] inside a (different) [list][2]. However, I still think this is the right approach.

[1]: https://github.com/googleapis/googleapis/blob/050400df0fdb16f63b63e9dee53819044bffc857/google/cloud/speech/v1beta1/cloud_speech.proto#L249
[2]: https://github.com/googleapis/googleapis/blob/050400df0fdb16f63b63e9dee53819044bffc857/google/cloud/speech/v1beta1/cloud_speech.proto#L394